bentoml/_internal/bento/docker/templates/_macros.j2

```diff
@@ -38,7 +38,7 @@ set -euxo pipefail
 
 if [ -f {{ path }} ]; then
   echo "{{ echo_msg }}..."
-  chmod +x {{ path }}
+  bash {{ path }}
   {{ path }}
```

Comment:
Seems to me like this line should be removed, and I vaguely recall seeing another chmod in this template somewhere.

Reply:
the chmod inside Docker are for entrypoint, not related.